src/components/ui/PageHeader.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const PageHeader = props => {
```

Comment:
Good job using prop-types but moving style to the bottom of the file could help in a style not getting our way while reading the code later for improvement